tutorials/how-to-use-ipfs/index.md

```diff
@@ -1 +1,114 @@
+In the decentralized world, we can consider Cosmos SDK chains as the backend of our decentralized applications (Dapps). The blockchain let us store and retrieve data. Dapps usually have a frontend as well, and that frontend can be stored in a centralized entity (AWS, GitHub Pages) or, for being 100% decentralized, use a protocol like IPFS.
+
+> IPFS is a distributed system for storing and accessing files, websites, applications, and data.
+
+This means that IPFS allows you to store your frontend in a decentralized manner.
+More information about how IPFS works on their [documentation](https://docs.ipfs.tech/concepts/what-is-ipfs).
+
+In this tutorial, we will learn how to store files and website on IPFS.
+
+## Requirements 
+
+First download and install IPFS, find the instructions for your OS [here](https://docs.ipfs.io/install/command-line/).
+
+Verify you have a working installation by running the following command:
+
+```bash
+> $ ipfs --version
+> ipfs version 0.15.0
+```
+
+## Submit a file
+
+To make a file available on IPFS, you need to make it available to an IPFS node.
+This can be a local IPFS node or a remote one. In this tutorial, we will use a remote node.
+
+For Cosmos chain governance, we want to store the metadata on IPFS nodes.
+Let's create a file called `metadata.json` with the following content:
+
+```json
+{
+  "title": "Funds alumni program",
+  "authors": "IDA Alumni",
+  "summary": "Lorem ipseum dolor sit amet",
+  "details": "",
+  "proposalForumURL": "https://forum.cosmos.network/t/foo",
+  "voteOptionContext": "",
+}
+```
+
+Add the file to your IPFS node:
+
+```bash
+$ ipfs add metadata.json
+added Qmbct5cv1SxMpEYCpNEvZnQ5TkN1rk7KyTmV95nbsrbLiR metadata.json
+```
+
+By running `add` you have [pinned the file](https://docs.ipfs.tech/how-to/pin-files/#three-kinds-of-pins) to your node, stopping it from being deleted.
+The content identifier (CID) of the file is `Qmbct5cv1SxMpEYCpNEvZnQ5TkN1rk7KyTmV95nbsrbLiR`. This is the CID that you will use to retrieve the file from IPFS.
+
+Verify the content of the file by running:
+
+```bash
+$ ipfs cat Qmbct5cv1SxMpEYCpNEvZnQ5TkN1rk7KyTmV95nbsrbLiR
+```
+
+At this point, we can start our local IPFS node:
+
+```bash
+$ ipfs deamon
+```
+
+The file will become available on IPFS. We can easily access it via `https://ipfs.io/ipfs/<cid>`, in our case `https://ipfs.io/ipfs/Qmbct5cv1SxMpEYCpNEvZnQ5TkN1rk7KyTmV95nbsrbLiR`.
+
+Note that as soon as you stop the node, it is possible that after a while, it won't be accessible on the network.
+This is because only your node has the file being pinned, other nodes can delete the file depending of their retention policy.
```

Comment:
```suggestion
Note that as soon as you stop the node, it is possible that after a while, the file will not be accessible on the network.
This is because only your node has pinned the file, other nodes may delete the file depending of their retention policy.
```